x-pack/plugins/enterprise_search/server/routes/app_search/engines.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import fetch from 'node-fetch';
+import querystring from 'querystring';
+import { schema } from '@kbn/config-schema';
+
+import { IRouteDependencies } from '../../plugin';
+import { ENGINES_PAGE_SIZE } from '../../../common/constants';
+
+export function registerEnginesRoute({ router, config, log }: IRouteDependencies) {
+  router.get(
+    {
+      path: '/api/app_search/engines',
+      validate: {
+        query: schema.object({
+          type: schema.oneOf([schema.literal('indexed'), schema.literal('meta')]),
+          pageIndex: schema.number(),
+        }),
+      },
+    },
+    async (context, request, response) => {
+      try {
+        const appSearchUrl = config.host as string;
```

Comment:
Small nit. This is actually `enterpriseSearchUrl` since it does not contain the `/as`

Reply:
Any thoughts about just removing the definition totally and doing
```const url = `${encodeURI(config.host)}/as/engines/collection?${params}`;```
I like that `appSearchUrl`/`enterpriseSearchUrl` is a bit more readable but maybe it's a bit repetitive

Reply:
Totes ok with that :+1:

Reply:
I stuck with `enterpriseSearchUrl` for now, we can def redefine later if it ends up feeling repetitive down the road!